extensions/change-case/package.json

```diff
@@ -72,6 +72,14 @@
       "label": "Pop to Root After Action",
       "required": true
     },
+    {
+      "name": "hideHUD",
+      "type": "checkbox",
+      "default": false,
+      "description": "Check this to hide the HUD after the action.",
+      "label": "Hide HUD After Action",
+      "required": true
+    },
```

Comment:
It's better to change the option name to `displayHUD` and default it to `true` if you want to make this option required.

Reply:
In general, we should be careful about making new preferences `required` since it will disturb users' workflow 😊